collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: review
+title: Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results
+tags: machine-learning, neural and evolutionary computing
+author: Tarvainen Antti, Valpola Harri
+cite:
+    authors: "Tarvainen et Valpola"
+    title:   "Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results"
+    venue:   "Neural Information Processing Systems 30 (NIPS 2017) pre-proceedings"
+pdf: https://arxiv.org/pdf/1703.01780.pdf
+---
+
+# Highlights
+
+- Method : The mean teachers model **averages model weigths**, instead of label predictions like the Temporal Ensembling method. Allows for more accurate targets and faster feedback between the student and teacher models.
+- Performances : demonstrates **enhanced performance on various datasets** (SVHN, CIFAR-10, and ImageNet) when combined with residual networks. Achieves lower error rates and higher accuracy compared to Temporal Ensembling, even when trained with fewer labeled examples.
+- Scalability and applicability : Enables working on **larger dataset**, and training with **fewer labeled examples**.
+
+# Introduction
+Limitations of deep learning models :
+- requires large number of parameters and is prone to overfitting. 
+- manual high quality labels for training data is expensive and time consuming.
+
+Interest in semi-supervised learning to use the unlabeled data effectively. 
+
+&rarr; focus on **regularization methods** in **semi-supervised learning**
+
+# Method 
+
+## Apllying noise and Ensembling method
+
+### **Noise regularization** : 
+- Add noise to the input or intermediate representations of a model
+- Helps learning abstract invariances
+- **Pushes decision boundaries away from labeled data points**  
+
+Limitations of noise regularization :
+- classification cost is **undefined for unlabeled examples** &rarr; not helpful in semi-supervised learning.
+
+&rarr; The $\Gamma$ model to overcome this
+
+### **The $\Gamma$ model [1]**
```

Comment:
The correct formatting of footnotes in Markdown requires a leading caret before the number (both inline and at the bottom of the page) +
Fix LaTeX mode (double dollar signs are needed in jekyll to enable LaTeX mode):
```suggestion
### **The $$\Gamma$$ model[^1]**
```